sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -698,17 +738,23 @@ def model_custom_sink(simplekv, KVs, final_table_name):
                   which can be used to perform further operations
               simplekv.open_table(access_token, table_name) -
                   creates a table named 'table_name'. Returns a table object.
-              simplekv.write_to_table(table, access_token, key, value) -
+              simplekv.write_to_table(access_token, table, key, value) -
                   writes a key-value pair to the given table.
               simplekv.rename_table(access_token, old_name, new_name) -
                   renames the table named 'old_name' to 'new_name'.
     KVs: the set of key-value pairs to be written in the example pipeline.
-    final_table_name: the prefix of final set of tables to be created by the
-                      example pipeline.
+    final_table_name_no_ptransform: the prefix of final set of tables to be
```

Comment:
Added more comments explaining this. Moved the comment describing the API to the top level.